samples/map/singleStepMap/README.md

```diff
@@ -0,0 +1,11 @@
+**Map**
```

Comment:
Rename this example to

* artithmetic_add

Reason: This example computes the addition of an arithmetic series.